src/Famix-Test6-Tests/FamixTest6Test.class.st

```diff
@@ -33,10 +33,6 @@ FamixTest6Test >> testDefaultValueOnCreation2 [
 
 	bacon eggs: 0.
 	self assert: bacon eggs equals: 0.
-	self assert: eggProperty defaultValue equals: 12.
-
-	bacon eggs: nil.
-	self assert: bacon eggs equals: nil.
```

Comment:
We argue that adding a default value is to avoid nil, so we remove this assertion from the test.